src/Blazored.Typeahead/Interop.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using System;
+using System.Threading.Tasks;
+
+namespace Blazored.Typeahead
+{
+    public static class Interop

```

Comment:
I think this should be internal, I don't think we need to expose this out to consumers.

Reply:
A class that needs `JSInvokable` has to be static, else the JS won't be able to call in it I think.

Reply:
Just tested it, has to be public.

Reply:
Of course, it does!